installer/src/orbic.rs

```diff
@@ -90,7 +90,7 @@ pub async fn install() -> Result<()> {
 
 pub async fn shell() -> Result<()> {
     println!(
-        "WARNING: The orbic USB installer is likely to go away in a future version of Rayhunter. Consider using ./installer util orbic-start-telnet instead."
+        "WARNING: The orbic USB installer is not recommended for most usecases. Consider using ./installer util orbic-shell instead, unless you want ADB access for other purposes."
```

Comment:
this was probably discussed while i was on vacation, but are we not planning on removing the USB installer? if we still are, might be worth leaving in language suggesting that it'll be deprecated

Reply:
Originally both the installer and the start-telnet command had the stronger language. It was pushed back upon during code review, and so the wording was changed to just "not recommended" in the installer. In this PR, I just updated the warning in the shell to align with the wording of the installer subcommand.

I think the objection wrt actually removing the USB installer was mainly

* that the network-based one requires an admin password and is therefore more difficult to use in mass-installation
* shell and file transfer are missing from the network installer

But I do still think we should set a goal to remove it. We might just have to do more work to get there.